.gitleaksignore

```diff
@@ -1 +0,0 @@
-018bcb19a754f9a34f0866ac247e89b2de1cf839:data-raw/app/calc_missing_address_paper_rx.R:nhsnumber:31
```

Comment:
I had some trouble with gitleaks (all false +ve's) at first. While trying to sort, I noticed this no longer needed ignore - the file referred to is no longer in the repo.